### ISSUE TITLE
add support for bitport.io

### ISSUE THREAD

**Issue by strahey** [rclone/rclone#5452]
#### The associated forum post URL from `https://forum.rclone.org`
https://forum.rclone.org/t/bitport-io-support/25220/2

#### What is your current rclone version (output from `rclone version`)?

v1.55.1

#### What problem are you are trying to solve?

i'm trying to automate a workflow that uses bitport.io as a torrent client with cloud storage which is very similar to put.io.  Bitport has a free level and a lower cost entry level than put.  i'm just trying to connect to bitport and download files in a specific folder to a specific local folder.  

#### How do you think rclone should be changed to solve that?
if rclone supported connecting to bitport.io and had copy, move, and sync commands would very helpful.  Thanks in advance for any consideration.  I'm using rclone for accessing gdrive and it has been fantastic.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Does bitport.io have an API? Can you find it and add it to the description? Thanks

**Comment by strahey** [comment]
Yes, i added the API Authentication and methods reference in the description. Thanks!

**Comment by mhk924** [comment]
Bump
